relay-ourlogs/src/otel_to_sentry.rs

```diff
@@ -59,6 +59,7 @@ pub fn otel_to_sentry_log(
         attributes,
         trace_id,
         span_id,
+        event_name,
         ..
```

Comment:
Any other fields hidden in `..` that why might want to transform? I personally like listing all the unused fields as `foo: _` to highlight that we ignore them deliberately.